dbms/src/Interpreters/Aggregator.h

```diff
@@ -970,7 +973,7 @@ class Aggregator
                const AggregateDescriptions & aggregates_,
                bool overflow_row_,
                const TiDB::TiDBCollators & collators_ = TiDB::dummy_collators)
-            : Params(Block(), keys_, aggregates_, overflow_row_, 0, OverflowMode::THROW, 0, 0, 0, false, "", collators_)
+            : Params(Block(), keys_, aggregates_, overflow_row_, 0, OverflowMode::THROW, 0, 0, 0, false, "", DEFAULT_BLOCK_SIZE, collators_)
```

Comment:
why use `DEFAULT_BLOCK_SIZE` here?

Reply:
> why use `DEFAULT_BLOCK_SIZE` here?

This constructor is no use, just set a default value for max_block_size.

Reply:
> why use `DEFAULT_BLOCK_SIZE` here?

done.